src/gui/splash.c

```diff
@@ -269,21 +284,21 @@ void darktable_splash_screen_set_progress_percent(const char *msg,
     char *text = g_strdup_printf(msg, percent);
     gtk_label_set_text(GTK_LABEL(progress_text), text);
     g_free(text);
-    gtk_widget_show(progress_text);
 
-    if(elapsed >= 2.0 && fraction > 0.001)
+    if(elapsed >= 2.0 && fraction > 0.02)
     {
       const double total = elapsed / fraction;
       const double remain = total - elapsed;
       const int minutes = remain / 60;
       const int seconds = remain - (60 * minutes);
-      char *rem_text = g_strdup_printf("⏲%4d:%02d", minutes, seconds);
+      char *rem_text = g_strdup_printf(" %4d:%02d", minutes, seconds);
       gtk_label_set_text(GTK_LABEL(remaining_text), rem_text);
       g_free(rem_text);
     }
     else
     {
-      gtk_label_set_text(GTK_LABEL(remaining_text), "⏲  --:--");
+      gtk_label_set_text(GTK_LABEL(remaining_text), "   --:--");
+      gtk_widget_hide(GTK_WIDGET(remaining_box));
```

Comment:
This 'hide' call prevents the clock from showing until the time estimate is available.  It will automatically be hidden on the next message update which doesn't include a time estimate.